### PR TITLE
Dynamic menu options

### PR DESCRIPTION
**Pull request by tmelliott**

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -527,181 +531,92 @@ shinyServer(function(input, output, session) {
...
+    
+    visualize_tabs = tabPanel("Visualize", value = "visualize", uiOutput("visualize.panel"))
+    row_ops_tabs = do.call("navbarMenu", c("Dataset", unname(row_ops_tabs)))
+    history_tabs = tabPanel("R code history", value = "rhistory", uiOutput("code.panel"))
+    
+    if(!is.null(session$userData$LITE_VERSION) &&
+       session$userData$LITE_VERSION == "CAS") {
+      import_tabs = import_tabs[names(import_tabs) != "export"]
+      manipulate_tabs = manipulate_tabs[names(manipulate_tabs) != "create"]
+      advance_tabs = advance_tabs[names(advance_tabs) %in% c("multiple", "multivariate")]
```
Comment (tmelliott) [inline server.R:595]: I guess drop to just the 'multiple' module - (it doesn't seem to be working, anyway).

Comment (chrk623) [reply server.R:595]: You mean don't include the multiple module inside `advance_tabs` in the frist place?

Actually.. `multivariate` is not included in the first place, so we dont need that in here either.

Comment (tmelliott) [reply server.R:595]: Vague on my part, sorry - keep "multiple" only.

Comment (chrk623) [reply server.R:595]: Will push this shortly. Will not affect the build as the multivariate tab was not included in the first place.

Comment (tmelliott) [reply server.R:595]: Cool - if you can, just update your 'dev' branch and push directly to that if it's just the one-line change.

**Pull request merged**